include/rocksdb/utilities/table_properties_collectors.h

```diff
@@ -172,12 +125,9 @@ class CompactForTieringCollectorFactory
   std::string ToString() const override;
 
  private:
-  mutable port::RWMutex rwlock_;  // synchronization mutex
-  std::map<uint32_t, size_t> compaction_triggers_;
-  bool enabled_;
+  std::atomic<double> compaction_trigger_ratio_;
```

Comment:
Recommend AcqRelAtomic, or even RelaxedAtomic as we don't need any ordering guarantees, just atomicity. (Sequential consistency has some non-trivial overheads, etc.)

Reply:
Yes, it's a great util. That is currently in an internal header though.